snuba/query/processors/logical/custom_function.py

```diff
@@ -39,6 +42,13 @@ def partial_function(body: str, constants: Sequence[Tuple[str, Any]]) -> Express
 
 class CustomFunction(LogicalQueryProcessor):
     """
+    WARNING
```

Comment:
Can you add a tech debt task in JIRA to track this?

Reply:
sure